### PR TITLE
fix: Top menu swap button, top menu coming soon, stand up footer, responsive staking screen

### PR DESCRIPTION
**Pull request by omniwired**
Fixed, it now goes to Swap.
image
Fixed: is not coming soon, but it shows the current existing daos.
image

### CODE REVIEW

**Comment by omniwired** [comment]
> This commit makes the `GovernanceScreen` layout wrong, wdyt ? => [9ef0332](https://github.com/TERITORI/teritori-dapp/commit/9ef0332cefe58fc6564d5192044ba2638145b20b)
> 
> image

updated with better styles

**Review by WaDadidou — CHANGES_REQUESTED** [review]

File: packages/components/ScreenContainer/index.tsx [hunk 1]
```diff
@@ -64,6 +65,16 @@ export const ScreenContainer: React.FC<{
   forceNetworkFeatures,
 }) => {
   const { height } = useWindowDimensions();
+  const navigation = useAppNavigation();
+
+  if (!onBackPress) {
```
Comment (hthieu1110) [inline packages/components/ScreenContainer/index.tsx:70]: IMHO we should not overide props. Suggestion:
```
const doBackPress = () => {
    onBackPress && navigation.canGoBack() && navigation.goBack();
}
```

File: packages/components/TopMenu/TopMenuMyTeritories.tsx [hunk 2]
```diff
@@ -1,161 +1,100 @@
...
-                </View>
-              )}
-            />
-            {hovered && <OrgsComingSoon />}
-          </>
-        )}
-      </CustomPressable>
+    <TopMenuSection title="All DAOs" isCarousel>
+      {daos && (
+        <>
```
Comment (hthieu1110) [inline packages/components/TopMenu/TopMenuMyTeritories.tsx:81]: no need <>

File: packages/screens/Governance/GovernanceDetails.tsx [hunk 3]
```diff
@@ -404,75 +375,53 @@ export const GovernanceDetails: React.FC<{
...
-                fontSize: 20,
-              }}
-            >
-              {totalParticipant}
-            </BrandText>
-          </View>
+        <View>
+          <BrandText
+            style={{
+              fontSize: 20,
```
Comment (hthieu1110) [inline packages/screens/Governance/GovernanceDetails.tsx:387]: Can we use : fontSemibold20 ?

File: packages/screens/Governance/GovernanceDetails.tsx [hunk 4]
```diff
@@ -404,75 +375,53 @@ export const GovernanceDetails: React.FC<{
...
-              { x: "Pourcentage Yes", y: valueChartYes },
-              { x: "Pourcentage No", y: valueChartNo },
-              { x: "Pourcentage Abstain", y: valueChartAbstain },
-            ]}
-            labels={() => null}
-          />
+        <View>
+          <BrandText
+            style={{
+              fontSize: 20,
```
Comment (hthieu1110) [inline packages/screens/Governance/GovernanceDetails.tsx:397]: can we use fontSemibold20 ?

File: packages/screens/Governance/GovernanceDetails.tsx [hunk 5]
```diff
@@ -495,111 +444,104 @@ export const GovernanceDetails: React.FC<{
...
+                width: 12,
+                height: 12,
+                backgroundColor: "#16BBFF",
+                borderRadius: 12,
+              }}
+            />
+            <BrandText
+              style={{
+                marginLeft: layout.spacing_x1,
+                fontSize: 14,
```
Comment (hthieu1110) [inline packages/screens/Governance/GovernanceDetails.tsx:469]: can we use fontSemibold14 or fontMedium14 ?

File: packages/screens/Governance/GovernanceDetails.tsx [hunk 6]
```diff
@@ -495,111 +444,104 @@ export const GovernanceDetails: React.FC<{
...
+              flexDirection: "row",
             }}
           >
-            NoWithVeto {percentageNoWithVeto}
-          </BrandText>
+            <View
+              style={{
+                width: 12,
+                height: 12,
+                backgroundColor: "#333333",
```
Comment (hthieu1110) [inline packages/screens/Governance/GovernanceDetails.tsx:531]: there is : neutral33 for this

**Review by hthieu1110 — DISMISSED** [review]
LGTM

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

File: packages/components/footers/Footer.tsx [hunk 7]
```diff
@@ -72,42 +70,6 @@ const FooterSocialNetworks: React.FC = () => {
   );
 };
 
-const FooterGetUp: React.FC = () => {
```
[the remark this answers is not in the document]
Comment (omniwired) [reply packages/components/footers/Footer.tsx:75]: yes

File: packages/screens/Marketplace/CollectionScreen.tsx [hunk 8]
```diff
@@ -59,7 +59,6 @@ export const CollectionScreen: ScreenFC<"Collection"> = ({ route }) => {
         </BrandText>
       }
       responsive
-      onBackPress={() => navigation.navigate("Marketplace")}
```
[the remark this answers is not in the document]
Comment (omniwired) [reply packages/screens/Marketplace/CollectionScreen.tsx:62]: fixed

File: packages/screens/Stake/StakeScreen.tsx [hunk 9]
```diff
@@ -70,7 +72,12 @@ export const StakeScreen: React.FC = () => {
   };
 
   return (
-    <ScreenContainer fullWidth forceNetworkKind={NetworkKind.Cosmos}>
+    <ScreenContainer
+      forceNetworkKind={NetworkKind.Cosmos}
+      headerChildren={<BrandText style={fontSemibold20}>Stake</BrandText>}
+      responsive
+      onBackPress={() => navigation.goBack()}
```
[the remark this answers is not in the document]
Comment (omniwired) [reply packages/screens/Stake/StakeScreen.tsx:79]: reverted

**Pull request closed without merging**